### DISCUSSION TITLE
Sort tree table by absolute values

### DISCUSSION THREAD

**Discussion by kascp** [atoti/atoti#818] [Q&A] [answered]
Hi
Is it possible to sort by absolute values in the tree table? Or is there any easy trick to get this done?
Thanks
Philipp

**Comment by tibdex** [comment]
Hi Philipp, can you give more details about your use-case and what you want to achieve? A screenshot would help.

**Comment by kascp** [comment]
Hi
See my screenshot below. This is a simplified example of a tree table I have in my atoti dashboard.
As you can see I sorted the column PROFIT SUM in ascending order (from most negative to most positive number). However, I would like to sort descending by absolute value, i.e. INSTRUMENT 100 should be ranked directly after INSTRUMENT 2.
Thank you
Philipp

<img width="333" alt="image" src="https://github.com/atoti/atoti/assets/53344988/89c549ff-39c5-4172-a2ba-eb120892da93">

**Reply by tibdex** [reply, marked as answer]
You could create a new measure and sort with it.

```python
m["ABS PROFIT SUM"] = atoti.math.abs(m["PROFIT SUM"])
```

**Reply by kascp** [reply]
thank you
